### PR TITLE
Unify Exception ToInvariantString method

### PR DESCRIPTION
**Pull request by Kielek**
Fixes #.

## Changes

Unify Exception ToInvariantString method. No more copy-paste same code.

* ~~[ ] Appropriate `CHANGELOG.md` updated for non-trivial changes~~
* ~~[ ] Design discussion issue #~~
* ~~[ ] Changes in public API reviewed~~

### CODE REVIEW

File: src/OpenTelemetry.Contrib.Shared/Api/ExceptionExtensions.cs [hunk 1]
```diff
@@ -28,18 +28,18 @@ internal static class ExceptionExtensions
     /// </summary>
     /// <param name="exception">Exception to convert to string.</param>
     /// <returns>Exception as string with no culture.</returns>
-    public static string ToInvariantString(this Exception exception)
+    public static string ToInvariantString(this System.Exception exception)
     {
-        var originalUICulture = Thread.CurrentThread.CurrentUICulture;
+        var originalUICulture = System.Threading.Thread.CurrentThread.CurrentUICulture;
```
Comment (Kielek) [inline src/OpenTelemetry.Contrib.Shared/Api/ExceptionExtensions.cs:33]: OneCollector is raising exception due to unneeded usings due to enabled `ImplicitUsings`.
It is the easiest way to fix this.

File: build/Common.props [hunk 2]
```diff
@@ -54,6 +54,10 @@
     <PackageReference Include="Microsoft.CodeCoverage" Version="$(MicrosoftCodeCoveragePkgVer)" Condition="'$(Configuration)'=='Release'" PrivateAssets="All" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(IncludeSharedExceptionExtensionSource)'=='true'">
```
Comment (utpilla) [inline build/Common.props:57]:
```suggestion
  <ItemGroup Condition="'$(IncludeSharedExceptionExtensionsSource)'=='true'">
```

Comment (Kielek) [reply build/Common.props]: Solved in 6cb3f09044d73cee3f3a3415204f1900497c80e4

**Review by utpilla — APPROVED** [review]

**Review by reyang — APPROVED** [review]
Nice improvement!

**Pull request merged**